.babelrc

```diff
@@ -10,6 +10,7 @@
     },
     "production": {
       "plugins": [
+        "transform-react-remove-prop-types",
```

Comment:
> If you ship two separate bundles, one to use for development and one to use in production

We could follow what react is doing, but that's out of the scope of this PR.

>  there's no point in wrapping the proptypes specification in an if (__DEV__)

No if you have the two pipelines utils. And because you are using React, you should have them anyway.
- replace `if (process.env.NODE_ENV !== "production") {` by `if (false)`
- have some dead code elimination tools that strip `if (false)`